config/routes.rb

```diff
@@ -20,6 +20,8 @@
   match "/auth/:provider/callback" => "auth#oauth_callback", via: [:get]
   get "/auth/failure" => "auth#failure"
   post "/logout" => "auth#logout"
+  resources :users, only: [:update]
```

Comment:
Sweet! +100 for the better security and standards-compliance this change gives us.